source/class/qx/tool/cli/api/CompilerApi.js

```diff
@@ -68,7 +93,7 @@ qx.Class.define("qx.tool.cli.api.CompilerApi", {
         config = await qx.tool.utils.Json.loadJsonAsync(compileJsonPath);
       }
       this.setConfiguration(config);
-      return config;
```

Comment:
this breaks existing compile.js files as like mine which relies on load returning the current config ... now it returns the result of the abstract load which is nothing  ... is there a purpose to this breakage ?

Reply:
It returns the promise of async load.
load could be async now - therefore can return a Promise.

Could you change to 
```
await load();
config = this.getConfiguration();
```